packages/react/cypress/component/auto/form/AutoFormDefaultValues.cy.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { api } from "../../../support/api.js";
+import { describeForEachAutoAdapter } from "../../../support/auto.js";
+
+describeForEachAutoAdapter("AutoForm", ({ name, adapter: { AutoForm }, wrapper }) => {
```

Comment:
Could you change the title to `AutoForm - default values` or something like that to indicate that this test runs stuff for default values (same for the global action test)